### ISSUE TITLE
[BUG] Unable to access Feathr.datasets in Databricks

### ISSUE THREAD

**Issue by amithadiraju1694** [feathr-ai/feathr#910]
### Willingness to contribute

No. I cannot contribute a bug fix at this time.

### Feathr version

0.9.0

### System information

- **OS Platform and Distribution (e.g., Linux Ubuntu 20.0)**: DBRT 10.4 LTS
- **Python version**: 3.8.1
- **Spark version, if reporting runtime issue**: 3.2.1

### Describe the problem

Unable to import feathr.datasets from any feathr package.

### Tracking information

Trying to run quick start demo from either [here](https://github.com/feathr-ai/feathr/blob/main/docs/samples/databricks/databricks_quickstart_nyc_taxi_demo.ipynb)  or [here](https://github.com/feathr-ai/feathr/blob/main/docs/samples/nyc_taxi_demo.ipynb). Unable to import `from feathr.datasets import nyc_taxi` it gives me `ModuleNotFoundError: No module named 'feathr.datasets'` in both tutorial notebooks.

I tried multiple ways to install feathr:

1. `pip install feathr[notebook]`
2. `pip install feathr==0.6.0` and `pip install feathr==0.9.0`
3. `pip install "git+https://github.com/feathr-ai/feathr#subdirectory=feathr_project"`

All methods give the same error as mentioned above.

### What component(s) does this bug affect?

- [X] `Python Client`: This is the client users use to interact with most of our API. Mostly written in Python.
- [ ] `Computation Engine`: The computation engine that execute the actual feature join and generation work. Mostly in Scala and Spark.
- [ ] `Feature Registry API`: The frontend API layer supports SQL, Purview(Atlas) as storage. The API layer is in Python(FAST API)
- [ ] `Feature Registry Web UI`: The Web UI for feature registry. Written in React

**Comment by loomlike** [comment]
Hi @amithadiraju1694 I just tested by myself and cannot reproduce the issue.

Here's what I did:
1. creating a new ADB cluster
2. install feathr via `pip install "git+https://github.com/feathr-ai/feathr#subdirectory=feathr_project"`, and 
3. I can import `from feathr.datasets import nyc_taxi`

If you already installed the feathr via `pip install feathr`, make sure you uninstall it or upgrade by using `-U` flag when installing from the github repo's main branch.
Also make sure you did "clear state" of the notebook before the testing.

Let me know if you still have an issue.
